src/test-utils/backend.ts

```diff
@@ -40,6 +42,7 @@ export const createExecutor = ({
     usb,
     fileSystem,
     dfu,
+    github: createGithubClient(config.startParams.githubToken),
```

Comment:
Why is this needed?

Reply:
It was during my tests, my was hitting the rate limit pretty often. Then I forgot about it.